qualtran/bloqs/arithmetic/comparison.py

```diff
@@ -884,11 +870,8 @@ def wire_symbol(self, reg: Optional[Register], idx: Tuple[int, ...] = tuple()) -
         raise ValueError(f'Unknown register symbol {reg.name}')
 
     def build_call_graph(self, ssa: 'SympySymbolAllocator') -> Set['BloqCountT']:
-        # TODO Determine precise clifford count and/or ignore.
-        # See: https://github.com/quantumlib/Qualtran/issues/219
-        # See: https://github.com/quantumlib/Qualtran/issues/217
         tc = t_complexity(LessThanConstant(self.bitsize, less_than_val=self.val))
-        return {(TGate(), tc.t)}
+        return {(TGate(), tc.t), (ArbitraryClifford(n=1), tc.clifford)}
```

Comment:
shouldn't this be just `return {(LessThanConstant(self.bitsize, less_than_val=self.val), 1)}`?

Reply:
Oh, I didn't know you could do that, but it makes sense.  I'll change it!